src/terra-dev-site/doc/how-to.2/show-loading-overlays.doc.mdx

```diff
@@ -0,0 +1,36 @@
+# How To Render Loading Overlays
+
+Loading overlays can be rendered within the framework by rendering an [ApplicationLoadingOverlay](/components/terra-application/reference/components/application-loading-overlay) component. Overlays will be rendered by the closest ancestor [ApplicationLoadingOverlayProvider](/components/terra-application/reference/components/application-loading-overlay).
+
+```jsx
```

Comment:
Can consumers leverage the closed `ApplicationLoadingOverlayProvider` and still create an overlay relative to it's container? Or will the overlay be overall the content under the provider or will it default to full screen?

Reply:
The location of the overlay entirely dependent upon the location of the ApplicationLoadingOverlayProvider in the component hierarchy. If a provider is presenting an overlay, it will overlay all content rendered within the provider.